### DISCUSSION TITLE
How to use Spatie Translatable with Relation Managers?

### DISCUSSION THREAD

**Discussion by tamasori** [filamentphp/filament#5051] [Help]
Hi!

I bumped into a problem. I need a relation manager for a resource. The resource is translatable, I am using the official Spatie Translatable package, and I need the relation manager to be Translatable as well. But it does not work. On The edit modal, it just shows [Object Object] for translatable fields. Is there a way, to make RelationManagers work with translations, or I have to create a whole resource for them?

Thank you in advanced!

**Comment by VladimirVM** [comment]
Some problem :(

**Comment by wejrcze** [comment]
Same issue here... it's annoying when you setup all and then get stuck on this little problem in relation manager got [Object Object] but in db it's correct please someone resolve this I cant use it any other way 👎

**Comment by broqit** [comment]
Up
